frontend/apps/app/features/migrations/pages/MigrationDetailPage/MigrationDetailPage.tsx

```diff
@@ -140,6 +155,55 @@ export const MigrationDetailPage: FC<Props> = async ({ migrationId }) => {
             {overallReview.reviewComment}
           </pre>
         </div>
+        <div className={styles.box}>
+          <h2 className={styles.h2}>Review Issues</h2>
+          <div className={styles.reviewIssues}>
+            {overallReview.reviewIssues.length > 0 ? (
+              [...overallReview.reviewIssues]
+                .sort((a, b) => {
+                  const severityOrder = {
+                    CRITICAL: 0,
+                    WARNING: 1,
+                    POSITIVE: 2,
+                  }
+                  return (
+                    severityOrder[a.severity as keyof typeof severityOrder] -
+                    severityOrder[b.severity as keyof typeof severityOrder]
+                  )
+                })
```

Comment:
Currently, the order is `CRITICAL -> `WARNING` -> `POSITIVE` , but it can be changed.